examples/haskell-stack/stack.yaml

```diff
@@ -0,0 +1,67 @@
+# This file was automatically generated by 'stack init'
```

Comment:
Ska vi checka in auto-generade filer?

Reply:
Ja om det rör sig om pakethantering etc, absolut säger jag. Kan inte Haskell men det här verkar jämförbart med npm och liknande, alltså package.json etc. https://stackoverflow.com/questions/64682066/should-stack-yaml-lock-be-checked-into-source-control